### PR TITLE
Support QuickSpec

### PR DESCRIPTION
**Pull request by toshi0383**
Support toggling between `Hoge.swift` and `HogeSpec.swift`.
I will rebase to master after #1 is merged.😉

### CODE REVIEW

File: xcode_toggleImplAndTests.scpt [hunk 1]
```diff
@@ -16,17 +16,27 @@ on run
...
 		# display dialog destinationName
 		
 		# ファイルパスを探して開く
-		set command to "/usr/bin/mdfind -onlyin " & quoted form of projectFolder & " -name " & quoted form of destinationName
 		# display dialog command
+		set command to "find " & quoted form of projectFolder & " -name " & quoted form of destinationName
 		set destinationPath to do shell script command
+		if length of destinationPath < 1 then
+			set destinationName to (my replaceThis:(w1 & ".*") inString:sourceName usingThis:w3)
+            display dialog destinationName
```
Comment (takasek) [inline xcode_toggleImplAndTests.scpt]: Oh, 4-space tabs mixed!

File: xcode_toggleImplAndTests.scpt [hunk 2]
```diff
@@ -16,17 +16,27 @@ on run
...
 			set destinationName to (my replaceThis:(w1 & ".*") inString:sourceName usingThis:w2)
 		end if
 		# display dialog destinationName
 		
 		# ファイルパスを探して開く
-		set command to "/usr/bin/mdfind -onlyin " & quoted form of projectFolder & " -name " & quoted form of destinationName
 		# display dialog command
+		set command to "find " & quoted form of projectFolder & " -name " & quoted form of destinationName
 		set destinationPath to do shell script command
+		if length of destinationPath < 1 then
```
Comment (takasek) [inline xcode_toggleImplAndTests.scpt]: This logic is somewhat unstructural.
If you can, using array to compose possible `destinationNames` makes the logic simple.  What do you think?
ちなみに I had tried to use the array but it doesn't work fine, so I abandoned...

**Comment by toshi0383** [comment]
Rebased to master and removed `display` lines.
Yea I think the logic is getting a little bit overwhelming. But I think we should move forward and merge anyways and leave refactoring as a future task. (I have to google it to figure out how to create an array in AppleScript!😇)

**Comment by takasek** [comment]
>we should move forward and merge anyways and leave refactoring as a future task.

I got it. Exactly!

**Pull request merged**